notebooks/rnnt_example_usage.ipynb

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
It can be - although it does contain quite a lot of information about how the new callbacks should be used. In particular, it points users towards using the `RNNTTraining` callback

Reply:
update: this notebook can now be deleted as the necessary callback (now named `TransducerForward` is automatically added).